### PR TITLE
Feature/cbcr 870

### PR DESCRIPTION
**Pull request by brian-digital**

### CODE REVIEW

File: app/uk/gov/hmrc/cbcrfrontend/controllers/StartController.scala [hunk 1]
```diff
@@ -38,7 +38,7 @@ import scala.concurrent.Future
 @Singleton
 class StartController @Inject()(val messagesApi: MessagesApi,
                                 val authConnector:AuthConnector)(implicit val cache:CBCSessionCache,
-                                                                 val auth:AuthConnector,
+//                                                                 val auth:AuthConnector,
```
Comment (peteanning) [inline app/uk/gov/hmrc/cbcrfrontend/controllers/StartController.scala]: Comment left in file?

File: project/FrontendBuild.scala [hunk 2]
```diff
@@ -28,14 +28,14 @@ object FrontendBuild extends Build with MicroService {
 
   )
 
-  def test(scope: String = "test") = Seq(
-    "uk.gov.hmrc" %% "hmrctest" % "3.0.0" % scope,
+  def test(scope: String = "test,it") = Seq(
+    "uk.gov.hmrc"             %% "hmrctest"             % "3.0.0" % scope,
 //    "org.scalatest" %% "scalatest" % "2.2.6" % scope,
```
Comment (peteanning) [inline project/FrontendBuild.scala]: Commented out dependency

File: test/uk/gov/hmrc/cbcrfrontend/controllers/SharedControllerSpec.scala [hunk 3]
```diff
@@ -30,10 +30,11 @@ import org.scalatestplus.play.guice.GuiceOneAppPerSuite
...
 import play.api.http.Status
-import play.api.i18n.MessagesApi
+import play.api.i18n.{Messages, MessagesApi}
 import play.api.libs.json.{JsValue, Json}
 import play.api.mvc.Result
 import play.api.test.FakeRequest
+import play.api.test.Helpers.contentAsString
 import uk.gov.hmrc.auth.core.{AffinityGroup, AuthConnector}
 import uk.gov.hmrc.cbcrfrontend.config.FrontendAppConfig
 //import uk.gov.hmrc.cbcrfrontend.controllers.auth.{TestSecuredActions, TestUsers}
```
Comment (peteanning) [inline test/uk/gov/hmrc/cbcrfrontend/controllers/SharedControllerSpec.scala]: commented line

File: app/uk/gov/hmrc/cbcrfrontend/controllers/StartController.scala (hunk 1 above)
Comment (brian-digital) [reply app/uk/gov/hmrc/cbcrfrontend/controllers/StartController.scala]: removed commented code

File: project/FrontendBuild.scala (hunk 2 above)
Comment (brian-digital) [reply project/FrontendBuild.scala]: removed commented code

File: test/uk/gov/hmrc/cbcrfrontend/controllers/SharedControllerSpec.scala (hunk 3 above)
Comment (brian-digital) [reply test/uk/gov/hmrc/cbcrfrontend/controllers/SharedControllerSpec.scala]: removed commented code

**Pull request merged**